push-build.sh

```diff
@@ -106,22 +108,22 @@ RELEASE_BUCKET=${FLAGS_bucket:-"kubernetes-release-dev"}
 # This will canonicalize the path
 KUBE_ROOT=$(pwd -P)
 
-KUBECTL_OUTPUT=$(cluster/kubectl.sh version --client 2>&1 || true)
-if [[ "$KUBECTL_OUTPUT" =~ GitVersion:\"(${VER_REGEX[release]}(\.${VER_REGEX[build]})?(-dirty)?)\", ]]; then
+gitlib::get_version_vars
+if [[ "$KUBE_GIT_VERSION" =~ (${VER_REGEX[release]}(\.${VER_REGEX[build]})?(-dirty)?) ]]; then
   LATEST=${BASH_REMATCH[1]}
-  if ((FLAGS_ci)) && [[ "$KUBECTL_OUTPUT" =~ GitTreeState:\"dirty\" ]]; then
+  if ((FLAGS_ci)) && [[ "$KUBE_GIT_TREE_STATE" =~ "dirty" ]]; then
     logecho "Refusing to push dirty build with --ci flag given."
     logecho "CI builds should always be performed from clean commits."
     logecho
-    logecho "kubectl version output:"
-    logecho $KUBECTL_OUTPUT
+    logecho "git version output:"
+    logecho KUBE_GIT_VERSION
     common::exit 1
   fi
 else
   logecho "Unable to get latest version from build tree!"
   logecho
-  logecho "kubectl version output:"
-  logecho $KUBECTL_OUTPUT
+  logecho "git version output:"
+  logecho KUBE_GIT_VERSION
```

Comment:
I assume this is meant to return the $value?

Reply:
Same as previous comment.